### PR TITLE
Add nu-search and upgrade elastic

### PR DESCRIPTION
**Pull request by samleeflang**
- Add files needed for coll, the job to generate the index and a resolver to download and resolve with
- Upgraded elastic a couple days ago + added some extra disk space to handle logs and monitoring

### CODE REVIEW

File: test/elastic/elasticsearch.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spec:
               - ReadWriteOnce
             resources:
               requests:
-                storage: 40Gi
+                storage: 80Gi
```
Comment (TomDijkema) [inline test/elastic/elasticsearch.yaml:21]: Noice

File: test/nu-search/nu-search-deployment.yaml [hunk 2]
```diff
@@ -51,16 +64,27 @@ spec:
...
+              ephemeral-storage: "10Gi"
           securityContext:
             runAsNonRoot: true
             allowPrivilegeEscalation: false
           volumeMounts:
             - mountPath: /index-files
               name: temp-volume
+            - name: aws-secrets
+              mountPath: "/mnt/secrets-store/aws-secrets"
+              readOnly: true
```
Comment (TomDijkema) [inline test/nu-search/nu-search-deployment.yaml:81]: Is this volume storing secrets or how does this manage?

File: test/nu-search/nu-search-deployment.yaml [hunk 3]
```diff
@@ -19,11 +19,24 @@ spec:
...
         co.elastic.logs/multiline.match: after
     spec:
+      serviceAccountName: nu-search-sa
       containers:
         - name: dissco-nu-search-deployment
           image: public.ecr.aws/dissco/dissco-nusearch-service
           imagePullPolicy: Always
           env:
+            - name: spring.profiles.active
+              value: s3-resolver
```
Comment (southeo) [inline test/nu-search/nu-search-deployment.yaml:29]: Do we need a deployment for the standalone profile as well?

**Review by southeo — APPROVED** [review]

File: test/nu-search/nu-search-deployment.yaml (hunk 3 above)
Comment (samleeflang) [reply test/nu-search/nu-search-deployment.yaml:29]: No not for dissco. The standalone version first downloads the ColDP and then indexes it. This is doable with small datasets but for larger datasets it requires a lot of memory and the startup time is long. With the S3 profiles we decoupled these two steps. So S3_Indexer indexes the dataset and puts it in an S3 bucket, this needs a low of memory but only needs it for +/- 30 minutes. The S3_Resolver then downloads the index from S3 and needs only a third of the original memory (was hoping for less, but apparently the S3 client doesn't stream download) and the startup takes round 20-30 sec.

**Pull request merged**